rewards/README.md

```diff
@@ -4,40 +4,80 @@ cover: ../.gitbook/assets/cover/rewards_banner.png
 coverY: 0
 ---
 
-# 💰 Rewards
+## What is Data Farming?
 
-### Why did we create the Data Farming dApp?
+Data Farming is Ocean Protocol's core incentive program that rewards participants to onboard the protocol and create value by using tools that help grow it further.
 
-The purpose of Ocean Protocol's Data Farming dApp reward system is to incentivize the curation and publishing of high-quality data NFTs in the Ocean Ecosystem. Data Farming participants earn OCEAN rewards for these activities. At a minimum, Data Farmers earn "passive rewards" for locking their OCEAN tokens to get veOCEAN tokens in return. Then, Data Farmers can maximize their yield by earning "active rewards" using their veOCEAN tokens. Active rewards are generated by participants voting on their favorite Ocean ecosystem NFTs by allocating their veOCEAN to these assets, thus gaining a portion of these asset sales.
+Data Farming participants earn OCEAN rewards across 2 main streams **Passive Rewards** and **Active Rewards**. Participants are able to **receive rewards passively, actively, or both** by engaging in a variety of activities.
+
+Passive Rewards provides a way for all contributors to participate.
+
+Active Rewards focuses on incentivizing the curation and publishing of high-quality data assets and DataNFTs within the Ocean Ecosystem. 
+
+We have used battle-tested contracts that safeguard billions (veCRV). With this model there is no liquidity risk. You can’t lose your OCEAN through Impermanent-Loss (IL). It is merely locked.
+
+## Introduction
+
+[Farming 101](df-intro.md) introduces Data Farming, the various reward systems, how they work, and how to access them. By the end of the document, you should feel more familiar with Data Farming concepts and having a better idea of what subject you might want to learn about next.
 
 ## veOCEAN
 
-Learning about [veOCEAN](veocean.md) will help you answer the question "What is the purpose of holding veOCEAN?" & give insights on how veOCEAN (vote-escrowed OCEAN) works. It will teach you everything you need to know about why it exists and how it works.
+Learning about [veOCEAN](df-veocean.md) will help you answer the question "What is the purpose of veOCEAN?" and give insights on how veOCEAN (vote-escrowed OCEAN) is used inside of Ocean Protocol to help advance the protocol.
```

Comment:
Something is off with this file, all the other links are working as expected, but df-veocean.md links are not correctly working. I'm investigating as well..